lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -148,7 +149,12 @@ std::string mapLLVMTypeToOCLType(const Type *Ty, bool Signed) {
     Ss << mapLLVMTypeToOCLType(EleTy, Signed) << Size;
     return Ss.str();
   }
-  return "invalid_type";
+  // Just mangle the type
```

Comment:
Some comment (or even an `assert`) about which types we expect here would be helpful

Reply:
Added a comment in e549d01